We cannot compare against origin/main branch because we build from sdist, not git.

--- pipdeptree-2.13.1/tox.ini.orig
+++ pipdeptree-2.13.1/tox.ini
@@ -30,7 +30,6 @@
       --cov-report html:{envtmpdir}{/}htmlcov --cov-report xml:{toxworkdir}{/}coverage.{envname}.xml \
       --junitxml {toxworkdir}{/}junit.{envname}.xml \
       tests}
-    diff-cover --compare-branch {env:DIFF_AGAINST:origin/main} {toxworkdir}{/}coverage.{envname}.xml --fail-under 100
 
 [testenv:fix]
 description = format the code base to adhere to our styles, and complain about what we cannot do automatically
--- pipdeptree-2.13.1/pyproject.toml.orig
+++ pipdeptree-2.13.1/pyproject.toml
@@ -44,7 +44,6 @@
 ]
 optional-dependencies.test = [
   "covdefaults>=2.3",
-  "diff-cover>=7.7",
   "pip>=23.2.1",
   "pytest>=7.4",
   "pytest-cov>=4.1",
